From ad22fb281de2be2c59e4cfb22ae7c22ac382a4bc Mon Sep 17 00:00:00 2001
From: Roy Li <rongqing.li@windriver.com>
Date: Fri, 21 Feb 2014 09:04:33 +0800
Subject: [PATCH] Give ppp_on_boot script correct file context

Upstream-Status: Inappropriate [only for Poky]

ppp_on_boot is a script, which will be called by /etc/init.d/ppp,
so they should have the same file context

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 policy/modules/contrib/ppp.fc |    1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/contrib/ppp.fc b/policy/modules/contrib/ppp.fc
index efcb653..98e91ff 100644
--- a/policy/modules/contrib/ppp.fc
+++ b/policy/modules/contrib/ppp.fc
@@ -8,6 +8,7 @@ HOME_DIR/\.ppprc	--	gen_context(system_u:object_r:ppp_home_t,s0)
 /etc/ppp/.*secrets	--	gen_context(system_u:object_r:pppd_secret_t,s0)
 /etc/ppp/resolv\.conf	--	gen_context(system_u:object_r:pppd_etc_rw_t,s0)
 /etc/ppp/(auth|ip(v6|x)?)-(up|down)	--	gen_context(system_u:object_r:pppd_initrc_exec_t,s0)
+/etc/ppp/ppp_on_boot	--	gen_context(system_u:object_r:pppd_initrc_exec_t,s0)
 
 /sbin/ppp-watch	--	gen_context(system_u:object_r:pppd_exec_t,s0)
 /sbin/pppoe-server	--	gen_context(system_u:object_r:pppd_exec_t,s0)
-- 
1.7.10.4

